src/plugins/controls/public/control_group/editor/create_control.tsx

```diff
@@ -33,6 +26,11 @@ export interface CreateControlButtonProps {
   closePopover?: () => void;
 }
 
+interface CreateControlPromise {
```

Comment:
Nit: I would call this `CreateControlResult` because the type `Promise<T>` creates a promise where the _result_ is typed as T.